.ci/jenkins/Jenkinsfile.buildchain

```diff
@@ -54,6 +54,21 @@ pipeline {
                 }
             }
         }
+        stage('Install build-chain tool') {
+            steps {
+                script {
+                    println "[INFO] Getting build-chain version from composite action file"
+                    def buildChainVersion = buildChain.getBuildChainVersionFromCompositeActionFile()
+                    if([null, 'null'].contains(buildChainVersion)) {
+                        def errorMessage = "[ERROR] The build-chain version can't be recovered. Please contact administrator"
+                        println errorMessage
+                        error(errorMessage)
+                    }
+                    println "[INFO] build-chain version recovered '${buildChainVersion}'"
+                    sh "npm install -g @kie/build-chain-action@${buildChainVersion} -registry=${NPM_REGISTRY_URL}"
```

Comment:
Not sure if `npm -g` requires sudo?

Reply:
@winklerm i guess it depends on the user of GHA, but great question

Reply:
@winklerm you can check the testing job https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/main/job/pullrequest/job/lienzo-core-main.pr/21/console and it is working without it. You can normally do it with or without `sudo` and the libraries will be placed in different places depending on this .
see 
```
$ npm get prefix
~/.npm
$ sudo npm get prefix
/usr
````

Reply:
it is true too, if user is always the same, it goes to NPM global user home dir